learning/training.py

```diff
@@ -237,9 +244,9 @@ def train_model(dls: DataLoaders, args: Namespace, run, rep: int):
 
     wandb_name = args.wandb_name
     model_arch = args.model_arch
-    dataset_name = args.dataset_name
+    dataset_names = "-".join(args.dataset_names)
 
-    learn_name = f"{wandb_name}-{model_arch}-{dataset_name}-rep{rep:02}"
+    learn_name = f"{wandb_name}-{model_arch}-{dataset_names}-rep{rep:02}"
```

Comment:
Do you think we need the dataset names as part of the file name? Maybe `wandb_name` is enough?

Reply:
It could be quicker to include it for documenting purposes if we produce more datasets later on. It would be quicker to have them automatically included without having to write them in the wandb_name argument. I went with it since it was always there but it's up to you.